### ISSUE TITLE
Add Connect code generator into vaadin-maven-plugin

### ISSUE THREAD

**Issue by vlukashov** [vaadin/flow#6239]
The following goals from the connect maven plugin should be integrated into the Vaadin maven plugin ```xml

<goal>generate-openapi-spec</goal>

<goal>generate-vaadin-client</goal>

<goal>generate-connect-modules</goal>
```

depends on #6222

**Comment by vlukashov** [comment]
merged into #6222
